modules/olm-mirroring-package-manifest-catalog.adoc

```diff
@@ -111,14 +112,14 @@ In this example, if you only want to mirror these images, you would then remove
 ----
 $ oc image mirror \
     [-a ${REG_CREDS}] \
-    -f ./redhat-operators-manifests/mapping.txt
+    -f ./redhat-operators-manifests-<random_number>/mapping.txt
 ----
 
 . Apply the `ImageContentSourcePolicy` object:
 +
 [source,terminal]
 ----
-$ oc apply -f ./redhat-operators-manifests/imageContentSourcePolicy.yaml
+$ oc apply -f ./redhat-operators-manifests-<random_number>/imageContentSourcePolicy.yaml
```

Comment:
It should be -f `./manifests-redhat-operators-<random_number>/imageContentSourcePolicy.yaml`.